sites/fast-website/src/public/index.html

```diff
@@ -9,6 +9,24 @@
     <fast-design-system-provider use-defaults>
       <fast-button>Hello, World!</fast-button>
       <gradient-picker>stub</gradient-picker>
+      <fast-section-header
+        short="Features & Utilities"
+        body="At id vitae iaculis volutpat laoreet. Massa ut scelerisque ut ac, in aliquam odio. Semper sagittis blandit in sed turpis lorem at diam iaculis."
+      >
+        <h5 slot="heading">A COMPREHENSIVE TOOLKIT</h5>
+        <fast-button slot="button" appearance="outline">Read documentation</fast-button>
+      </fast-section-header>
     </fast-design-system-provider>
+
+    <style>
+      fast-section-header h5 {
+        margin: 0;
+      }
+      fast-section-header::part(short) {
+        font-size: 46px;
+        line-height: 50px;
+        margin: 0;
+      }
+    </style>
```

Comment:
@radium-v @eljefe223 This is likely going to get fairly convoluted fairly quickly. Would it make sense to add storybook as a dev dependency so we can develop these in isolation more efficiently? It seems like some of these controls may have different states or variations and that might be difficult to keep track of in a monolithic html file over time. Just a thought.

Reply:
There's an added stylesheet in #3068 and I plan on adding support for CSS splitting to the webpack build, so these style chunks can sit with their respective components.

Reply:
Should each component utilize the existing patterns we have in `fast-components`? We've built-in some fairly good performance optimizations there and we can leverage the actual typography as well. Super simple example: https://github.com/microsoft/fast-dna/tree/master/packages/web-components/fast-components/src/card

Reply:
They definitely should follow the existing patterns.